backend/src/apiserver/filter/filter.go

```diff
@@ -207,6 +209,32 @@ func replaceMapKeys(m map[string][]interface{}, keyMap map[string]string, prefix
 	return nil
 }
 
+func (f *Filter) FilterK8sPipelines(pipeline v2beta1.Pipeline) bool {
+	// TODO: For now k8s pipelines filter will only work on substrings
+	for k := range f.substring {
+		for _, v := range f.substring[k] {
+			if strings.Contains(fmt.Sprint(pipeline.GetField(k)), fmt.Sprint(v)) {
+				return true
+			}
+		}
+	}
+
+	return false
+}
+
+func (f *Filter) FilterK8sPipelineVersions(pipelineVersion v2beta1.PipelineVersion) bool {
+	// TODO: For now k8s pipelines filter will only work on substrings
```

Comment:
the common pattern in the codebase is to put your github handle the todo comments in parentheses  (applies to all the todo comments being added with this pr)

Reply:
Although I agree this could be a good practice, I don't think this is needed. After all, there are many TODO comments throughout the KFP code with no github handles assigned to. However, it is possible to check the owner of that TODO comment with a `git blame`. Example: https://github.com/kubeflow/pipelines/blame/master/backend/src/v2/driver/driver.go#L2272

I will add my GH handle in all my TODO comments, but we should discuss if we really need to adopt this pattern.